ambry-router/src/main/java/com.github.ambry.router/NonBlockingRouterMetrics.java

```diff
@@ -491,21 +496,21 @@ private void initializeResourceToHistogramMap(ClusterMap clusterMap, RouterConfi
     switch (routerConfig.routerOperationTrackerMetricScope) {
       case Partition:
         for (PartitionId partitionId : clusterMap.getAllPartitionIds(null)) {
-          getBlobLocalDcResourceToLatency.put(partitionId, createHistogram(reservoirSize, decayFactor));
-          getBlobInfoLocalDcResourceToLatency.put(partitionId, createHistogram(reservoirSize, decayFactor));
-          getBlobCrossDcResourceToLatency.put(partitionId, createHistogram(reservoirSize, decayFactor));
-          getBlobInfoCrossDcResourceToLatency.put(partitionId, createHistogram(reservoirSize, decayFactor));
+          getBlobLocalDcResourceToLatency.put(partitionId, createHistogram(routerConfig, false));
```

Comment:
minor: local variables `reservoirSize` and `decayFactor` can be removed (line493,494)